stdlib/Makefile

```diff
@@ -0,0 +1,28 @@
+default: get
+
+PKG3_VERSION := a554607c939ab73d89c3e7c6a3dd08fd870145e8
+PKG3_URL := https://github.com/JuliaLang/Pkg3.jl
```

Comment:
This looks reasonable to me.  It's annoying to be in yet another bootstrap situation where the tooling (Pkg3 in this case) can't be used to build itself.

What would presumably be easier would be for an already built version of Pkg3 to emit a consistent set of requirements for the stdlib which could then be hard coded in this repo in a format which this makefile can consume.

Then updating this will be less manual as the stdlib grows to include more modules from various repositories.

Reply:
(caveat: I know next to nothing about `Pkg3`!)